### PR TITLE
docs: remove deprecated `loadNuxt` options

### PR DESCRIPTION
**Pull request by MichaelJCole**
### 🔗 Linked issue

https://github.com/nuxt/nuxt/issues/24197

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Update loadNuxt documentation to latest version.  `config` option was removed.  `cwd` option is standard.
The docs are out of date for these options.  I filed a bug, then found the fix.  This updates the docs for the next guy.

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/3.api/5.kit/2.programmatic.md [hunk 1]
```diff
@@ -53,25 +53,21 @@ Loading conditions for Nuxt. `loadNuxt` uses [`c12`](https://github.com/unjs/c12
   
   If set to `true`, Nuxt will be ready to use after the `loadNuxt` call. If set to `false`, you will need to call `nuxt.ready()` to make sure Nuxt is ready to use.
 
-- `rootDir` (optional)
+- `cwd` (optional)
 
   **Type**: `string`
 
   **Default**: `null`
```
Comment (DamianGlowala) [inline docs/3.api/5.kit/2.programmatic.md:60]:
```suggestion
  **Default**: `undefined`
```

Comment (DamianGlowala) [inline docs/3.api/5.kit/2.programmatic.md:60]:
```suggestion
  **Default**: `undefined`
```

File: docs/3.api/5.kit/2.programmatic.md [hunk 2]
```diff
@@ -53,25 +53,21 @@ Loading conditions for Nuxt. `loadNuxt` uses [`c12`](https://github.com/unjs/c12
...
 
-  **Deprecated**: Use `cwd` option instead.
+  The root directory of the Nuxt project.  The alias `rootDir` has been **Depricated**
```
Comment (DamianGlowala) [inline docs/3.api/5.kit/2.programmatic.md:62]:
```suggestion
  The root directory of the Nuxt project.
```

File: docs/3.api/5.kit/2.programmatic.md [hunk 3]
```diff
@@ -53,25 +53,21 @@ Loading conditions for Nuxt. `loadNuxt` uses [`c12`](https://github.com/unjs/c12
...
+- `overrides` (optional)
 
   **Type**: `LoadNuxtConfigOptions['overrides']`
 
   **Default**: `{}`
 
-  **Deprecated**: Use `overrides` option instead.
-
-  Overrides for the Nuxt configuration.
+  Overrides for the Nuxt configuration.  The alias `config` has been **Removed**
```
Comment (DamianGlowala) [inline docs/3.api/5.kit/2.programmatic.md:70]:
```suggestion
  Overrides for the Nuxt configuration.
```

**Review by DamianGlowala — CHANGES_REQUESTED** [review]

File: docs/3.api/5.kit/2.programmatic.md [hunk 4]
```diff
@@ -53,24 +53,20 @@ Loading conditions for Nuxt. `loadNuxt` uses [`c12`](https://github.com/unjs/c12
   
   If set to `true`, Nuxt will be ready to use after the `loadNuxt` call. If set to `false`, you will need to call `nuxt.ready()` to make sure Nuxt is ready to use.
 
-- `rootDir` (optional)
+- `cwd` (optional)
```
Comment (danielroe) [inline docs/3.api/5.kit/2.programmatic.md:56]: These two parameters are both documented in the link above (https://github.com/unjs/c12).

This is meant to be a list of _additional_ options that are respected.

Instead, shall we just remove them?

**Pull request merged**